### ISSUE TITLE
How to change the size of in-game scene images

### ISSUE THREAD

**Issue by yuyu-2021** [Albeoris/Memoria#832]
Hello, we are a team for the Chinese localization mod . Very grateful for the efforts made by Memoria. It is your efforts that give us a better gaming experience. Respect!

lately one of our team members asked for your help about replacing the language symbol on the title interface . And it is well soluted , it is really of great help and thank you so much!

Now here's a another problem.

About the in-game scene images, we have found the layout data for the them and we successfully replaced the Japanese images with Chinese by adjusting the grid positions .

Here is the display result:
![FF9 2024-09-11 09-14-04](https://github.com/user-attachments/assets/2cf5b381-59b7-4533-b0e1-60277e587cc5)

However, we are now encountering a new problem. Due to the current small image size, some of the localized text doesn't fit properly. (If we adjust the text length to fit it , it will compromise the aesthetic of the translated text.)

For example:
![8e85e724a3c38dfc38feee2137afb947](https://github.com/user-attachments/assets/941d1f08-308a-48a3-b57e-1cfe003e210a)

Another issue is that the Chinese text appears relatively small and it isn't very visually appealing. (Larger Chinese characters tend to look more impressive.)

Like this:
![FF9 2024-09-11 08-34-51](https://github.com/user-attachments/assets/6471787b-fed9-4a83-be3f-74edb6ca8467)

( Too small......)

Is it possible that we make use of this red-framed area to add more text?
![{E618A4DD-B0C9-B9E9-3FA6-7D0007CAB4E8}](https://github.com/user-attachments/assets/88d9be28-488f-4cc3-bd4e-54599968614b)

If it can be resolved, could you please let us know how we can use this feature in Memoria?（Including the "JP" icon on  on the title interface .）

**Comment by Tirlititi** [comment]
Hello.
About the "JP" icon on the title interface, it is done, as you have read it, but the solution will take effect only after the next Memoria release. It is normal if the line ``NameShort,...`` doesn't work already.

About the texts in the image ``Ending_Text_US_JP_GR_Atlas``, it can be done by specifying a "TPSHEET", as described here: [Sprites and atlases](https://github.com/Albeoris/Memoria/wiki/Sprites-and-atlases)
I see that there are some problems with that altas in some situations, but adding a ``.tpsheet`` might work for you already, depending on your setting. Here is the default one:
```txt
:format=40000
:size=2048x2048
ending_text_01_gr;0;1406;1068;84;0;0;27;26;26;42;0;0;0;0
ending_text_01_jp;1070;1409;900;81;0;0;111;110;31;40;0;0;0;0
ending_text_01_us;987;1793;840;100;0;0;144;137;26;26;0;0;0;0
ending_text_02_gr;0;1578;1210;84;0;0;27;28;27;41;0;0;0;0
ending_text_02_jp;1376;1323;666;81;0;0;38;561;31;40;0;0;0;0
ending_text_02_us;1212;1578;750;84;0;0;34;481;28;40;0;0;0;0
ending_text_03_gr;609;1304;765;100;0;0;27;33;26;26;0;0;0;0
ending_text_03_jp;0;559;573;79;0;0;215;37;32;41;0;0;0;0
ending_text_03_us;0;473;423;84;0;0;374;28;26;42;0;0;0;0
ending_text_04_gr;0;1492;1081;84;0;0;147;145;26;42;0;0;0;0
ending_text_04_jp;1083;1494;925;82;0;0;226;222;31;39;0;0;0;0
ending_text_04_us;0;1664;1087;100;0;0;144;142;26;26;0;0;0;0
ending_text_05_gr;1089;1672;865;92;0;0;36;28;25;38;0;0;0;0
ending_text_05_jp;1829;1816;201;77;0;0;31;697;34;44;0;0;0;0
ending_text_05_us;1847;1964;160;84;0;0;28;741;26;45;0;0;0;0
ending_text_06_gr;0;749;406;113;0;0;309;15;14;25;0;0;0;0
ending_text_06_jp;0;640;425;107;0;0;289;16;22;23;0;0;0;0
ending_text_06_us;0;1274;607;130;0;0;102;21;13;9;0;0;0;0
ending_text_07_bg_gr;0;1895;985;153;0;0;0;0;0;0;0;0;0;0
ending_text_07_bg_jp;987;1895;858;153;0;0;64;63;0;0;0;0;0;0
ending_text_07_bg_us;0;1134;412;138;0;0;286;287;8;7;0;0;0;0
ending_text_07_gr;0;1766;985;127;0;0;0;0;2;24;0;0;0;0
ending_text_07_jp;0;864;416;128;0;0;285;284;11;14;0;0;0;0
ending_text_07_us;0;994;402;138;0;0;292;291;8;7;0;0;0;0
```
The 3rd number of each line is the width that you want to change.
I'll make it so these atlases get exported as well with the option ``[Export] Graphics`` for the next update of Memoria.

About World Map continent titles and other place titles, I'll see what I can do in order for them to have any size you want and bypass the limitation due to background tiles.

Good luck!

**Comment by yuyu-2021** [comment]
> Hello. About the "JP" icon on the title interface, it is done, as you have read it, but the solution will take effect only after the next Memoria release. It is normal if the line `NameShort,...` doesn't work already.
> 
> About the texts in the image `Ending_Text_US_JP_GR_Atlas`, it can be done by specifying a "TPSHEET", as described here: [Sprites and atlases](https://github.com/Albeoris/Memoria/wiki/Sprites-and-atlases) I see that there are some problems with that altas in some situations, but adding a `.tpsheet` might work for you already, depending on your setting. Here is the default one:
> 
> ```
> :format=40000
> :size=2048x2048
> ending_text_01_gr;0;1406;1068;84;0;0;27;26;26;42;0;0;0;0
> ending_text_01_jp;1070;1409;900;81;0;0;111;110;31;40;0;0;0;0
> ending_text_01_us;987;1793;840;100;0;0;144;137;26;26;0;0;0;0
> ending_text_02_gr;0;1578;1210;84;0;0;27;28;27;41;0;0;0;0
> ending_text_02_jp;1376;1323;666;81;0;0;38;561;31;40;0;0;0;0
> ending_text_02_us;1212;1578;750;84;0;0;34;481;28;40;0;0;0;0
> ending_text_03_gr;609;1304;765;100;0;0;27;33;26;26;0;0;0;0
> ending_text_03_jp;0;559;573;79;0;0;215;37;32;41;0;0;0;0
> ending_text_03_us;0;473;423;84;0;0;374;28;26;42;0;0;0;0
> ending_text_04_gr;0;1492;1081;84;0;0;147;145;26;42;0;0;0;0
> ending_text_04_jp;1083;1494;925;82;0;0;226;222;31;39;0;0;0;0
> ending_text_04_us;0;1664;1087;100;0;0;144;142;26;26;0;0;0;0
> ending_text_05_gr;1089;1672;865;92;0;0;36;28;25;38;0;0;0;0
> ending_text_05_jp;1829;1816;201;77;0;0;31;697;34;44;0;0;0;0
> ending_text_05_us;1847;1964;160;84;0;0;28;741;26;45;0;0;0;0
> ending_text_06_gr;0;749;406;113;0;0;309;15;14;25;0;0;0;0
> ending_text_06_jp;0;640;425;107;0;0;289;16;22;23;0;0;0;0
> ending_text_06_us;0;1274;607;130;0;0;102;21;13;9;0;0;0;0
> ending_text_07_bg_gr;0;1895;985;153;0;0;0;0;0;0;0;0;0;0
> ending_text_07_bg_jp;987;1895;858;153;0;0;64;63;0;0;0;0;0;0
> ending_text_07_bg_us;0;1134;412;138;0;0;286;287;8;7;0;0;0;0
> ending_text_07_gr;0;1766;985;127;0;0;0;0;2;24;0;0;0;0
> ending_text_07_jp;0;864;416;128;0;0;285;284;11;14;0;0;0;0
> ending_text_07_us;0;994;402;138;0;0;292;291;8;7;0;0;0;0
> ```
> 
> The 3rd number of each line is the width that you want to change. I'll make it so these atlases get exported as well with the option `[Export] Graphics` for the next update of Memoria.
> 
> About World Map continent titles and other place titles, I'll see what I can do in order for them to have any size you want and bypass the limitation due to background tiles.
> 
> Good luck!

Appreciate so much.

The Chinese localization mod is 99.9999% done right now and I believe it will get completed by the releasing of the next version of memoria.

Really looking forward for that.

Respect again!
